go/src/koding/klient/terminal/terminal.go

```diff
@@ -253,7 +255,13 @@ func (t *Terminal) Connect(r *kite.Request) (interface{}, error) {
 	}
 
 	args = append(args, command.Args...)
-	cmd := exec.Command("/usr/bin/sudo", args...)
+	var cmd *exec.Cmd
+
+	if _, err := exec.LookPath("sudo"); err == nil {
```

Comment:
I will change it to os.Stat. I think the path must always be `/usr/bin/sudo`, executing it from different path can be a security threat I believe.